packages/sdk-codegen/src/python.gen.ts

```diff
@@ -86,6 +86,7 @@ export class PythonGen extends CodeGen {
     'in',
     'is',
     'lambda',
+    'models',
```

Comment:
Honestly I think we might be better off renaming the import. The plus side is we stay truer to the API spec. The downside is we break anyone who is doing something like `from looker_sdk.sdk.api40.methods import models` but they should be doing `from looker_sdk.sdk.api40 import models`

Reply:
I went with this, BTW.